demos/src/scss/_demos.scss

```diff
@@ -48,10 +48,9 @@ body:before {
 }
 
 .o-grid-container {
-	background: lighten(#f6e9d8, 8.5%);
+	background: #EAE8DF;
 }
```

Comment:
I understand your point, but this demo isn't for people that are working on the component. It's for users of the component, and our aim over the coming months will be to make using components a lot easier. Those that are working on the component can add a colour to the row for working/debugging should they need to quite easily.